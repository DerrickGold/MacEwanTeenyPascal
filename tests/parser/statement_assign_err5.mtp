begin
	(*Bad assignment, no value or assignment*)

	(*assignment statement: with factor (constant)*)
	something;
	
end
.