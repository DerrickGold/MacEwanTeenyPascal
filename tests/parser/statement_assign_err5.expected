Syntax Error: line 5:
Unexpected token (TOK_SEMICOLON, ';'), likely missing a(n)  
  'TOK_ASSIGN' or
  'TOK_LPAREN' or
Possibly due to malformed or missing:
  Non-terminal: 'Variable' or
  Non-terminal: 'Type'
